test/connectors/CustomsDeclareExportsConnectorIntegrationSpec.scala

```diff
@@ -62,9 +66,9 @@ class CustomsDeclareExportsConnectorIntegrationSpec
           )
       )
 
-      val response = await(connector.createDeclaration(newDeclaration))
```

Comment:
I can't find await method in scalatest, this one was from hmrc test

Reply:
`import play.api.test.Helpers._`

Reply:
ah, okay, I'll change it here :)

Reply:
Changed, waiting for build and merge